src/main/java/seedu/duke/exercise/ExerciseList.java

```diff
@@ -0,0 +1,76 @@
+package seedu.duke.exercise;
+
+import java.util.ArrayList;
+
+public class ExerciseList {
+
+    public static final String MESSAGE_EXERCISE_ADDED = "An exercise has been added:";
+    private static final String MESSAGE_EXERCISE_DELETED = "You have removed the exercise:";
+    private static final String MESSAGE_EMPTY_EXERCISE_LIST = "No exercise is found.";
+    private ArrayList<Exercise> exerciseList = new ArrayList<>();
+
+    /**
+     * Returns exercise item at a specific index in the exercise list.
+     *
+     * @param index Index of the exercise item.
+     * @return exercise item in the exercise list.
+     */
+    public Exercise getExercise(int index) {
+        return this.exerciseList.get(index);
+    }
+
+    /**
+     * Returns the number of exercises in the exercise list.
+     *
+     * @return Number of exercises in the exercise list.
+     */
+    public int getSize() {
+        return exerciseList.size();
+    }
+
+    /**
+     * Adds an exercise item into the exercise list.
+     *
+     * @param exercise Exercise class object to be added.
+     */
+    public void addExercise(Exercise exercise) {
```

Comment:
Maybe we can discuss and standardize the method name, whether to call it `add` or `addExercise` so that the add method in foodlist can follow the same style.

Reply:
yes I agree, with this and the delete and size methods. Also one thing to consider is that we should standardize our static final messages to be printed so that the whole app will look more coherent :) i have added a comment under your pull request regarding this